src/views/Ifos/components/IfoV1Card/ContributeModal.tsx

```diff
@@ -3,13 +3,13 @@ import { useWeb3React } from '@web3-react/core'
 import BigNumber from 'bignumber.js'
 import { ethers } from 'ethers'
 import { Modal, LinkExternal, Box, Text } from '@pancakeswap-libs/uikit'
-import BalanceInput from 'components/BalanceInput'
 import useTokenBalance from 'hooks/useTokenBalance'
 import { getBalanceNumber } from 'utils/formatBalance'
 import useI18n from 'hooks/useI18n'
 import ApproveConfirmButtons from 'views/Profile/components/ApproveConfirmButtons'
 import useApproveConfirmTransaction from 'hooks/useApproveConfirmTransaction'
 import { useERC20 } from 'hooks/useContract'
+import BalanceInput from './BalanceInput'
```

Comment:
Is this the same as the one in the UI Kit?

Reply:
Yes. It's the only place where it's still used. After the release, everything that is inside the V1 folder will be removed